nilearn/plotting/tests/test_html_surface.py

```diff
@@ -0,0 +1,329 @@
+    body = root.find('body')
+    div = body.find('div')
+    assert ('id', 'surface-plot') in div.items()
+    selects = body.findall('select')
+    assert len(selects) == 3
+    hemi = selects[0]
+    assert ('id', 'select-hemisphere') in hemi.items()
+    assert len(hemi.findall('option')) == 2
+    kind = selects[1]
+    assert ('id', 'select-kind') in kind.items()
+    assert len(kind.findall('option')) == 2
+    view = selects[2]
+    assert ('id', 'select-view') in view.items()
+    assert len(view.findall('option')) == 7
+
+
+def _open_mock(f):
+    print('opened {}'.format(f))
+
+
+def _check_open_in_browser(html):
+    wb_open = webbrowser.open
+    webbrowser.open = _open_mock
+    try:
+        html.open_in_browser()
+        temp_file = html._temp_file
+        assert html._temp_file is not None
+        assert os.path.isfile(temp_file)
+        html.remove_temp_file()
+        assert html._temp_file is None
+        assert not os.path.isfile(temp_file)
+        html.remove_temp_file()
+        html._temp_file = 'aaaaaaaaaaaaaaaaaaaaaa'
+        html.remove_temp_file()
+    finally:
+        webbrowser.open = wb_open
+        try:
+            os.remove(temp_file)
+        except Exception:
+            pass
+
+
+def test_fill_html_template():
+    fsaverage = fetch_surf_fsaverage()
+    mesh = surface.load_surf_mesh(fsaverage['pial_right'])
+    surf_map = mesh[0][:, 0]
+    img = _get_img()
+    info = html_surface.one_mesh_info(
+        surf_map, fsaverage['pial_right'], '90%', black_bg=True,
+        bg_map=fsaverage['sulc_right'])
+    html = html_surface._fill_html_template(info, embed_js=False)
+    _check_html(html)
+    assert "jquery.min.js" in html.html
+    info = html_surface.full_brain_info(img)
+    html = html_surface._fill_html_template(info)
+    _check_html(html)
+    assert "* plotly.js (gl3d - minified) v1.38.3" in html.html
+
+
+def test_view_surf():
+    fsaverage = fetch_surf_fsaverage()
+    mesh = surface.load_surf_mesh(fsaverage['pial_right'])
+    surf_map = mesh[0][:, 0]
+    html = html_surface.view_surf(fsaverage['pial_right'], surf_map,
+                                  fsaverage['sulc_right'], '90%')
+    _check_html(html)
+    html = html_surface.view_surf(fsaverage['pial_right'], surf_map,
+                                  fsaverage['sulc_right'], .3)
+    _check_html(html)
+    html = html_surface.view_surf(fsaverage['pial_right'])
+    _check_html(html)
+    destrieux = datasets.fetch_atlas_surf_destrieux()['map_left']
+    html = html_surface.view_surf(
+        fsaverage['pial_left'], destrieux, symmetric_cmap=False)
+    _check_html(html)
+    assert_raises(ValueError, html_surface.view_surf, mesh, mesh[0][::2, 0])
+    assert_raises(ValueError, html_surface.view_surf, mesh, mesh[0][:, 0],
+                  bg_map=mesh[0][::2, 0])
+
+
+def test_view_img_on_surf():
+    img = _get_img()
+    html = html_surface.view_img_on_surf(img, threshold='92.3%')
```

Comment:
I am sure it will work. But, could you add more arguments to these tests, like ```cmap```.